### PR TITLE
Add deploy-prometheus-grafana scenario in introduction

### PR DESCRIPTION
**Pull request by 4n4nd**
Create a pathway file for PAD Workshop

Can be tested [here](https://www.katacoda.com/4n4nd-debug/courses/ml-prometheus-metrics/deploy-prometheus-grafana)

### CODE REVIEW

File: introduction/deploy-prometheus-grafana/4create-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+![Grafana Create Panel Page](../../assets/introduction/deploy-prometheus-grafana/04-grafana-1-panel-dashboard-page.png)
+
+* In the upper right corner of the dashboard page, click on `Add Panel` and repeat the previous steps only this time
+use a different query. <br>
+`rate(flask_http_request_total[2m])`{{copy}} this should give you the rate of requests served by the application over 2 minutes.
+
+* Once you have set up the second panel, your dashboard should look something like this <br>
+![Grafana Create Panel Page](../../assets/introduction/deploy-prometheus-grafana/04-grafana-2-panel-dashboard-page.png)
+
+That's it! you should have your own monitoring dashboard now
```
Comment (hemajv) [inline introduction/deploy-prometheus-grafana/4create-dashboard.md]: maybe include a line about "saving" the panels/dashboard?

Comment (4n4nd) [reply introduction/deploy-prometheus-grafana/4create-dashboard.md]: Updated :+1:

**Comment by 4n4nd** [comment]
@harshad16 I think the bot is confused

**Review by saisankargochhayat — APPROVED** [review]
It seems to work fine on katacoda. And good tutorial :+1:

**Review by hemajv — APPROVED** [review]
:ship:

**Pull request merged**